src/main/webapp/components/BuildSnapshot.js

```diff
@@ -40,8 +41,11 @@ export const BuildSnapshot = (props) => {
   const headerData = {commitHash, description, repository, status};
   const trayData = {builders, timestamp};
 
+  // Set background based on build status.
+  const snapshotClassName = 'build-snapshot'.concat(' ' + status);
```

Comment:
this could be simplified a bit: it combines `concat` and `+` inside the argument
```suggestion
  const snapshotClassName = 'build-snapshot ' + status;
```